lib/Dialect/TTMetal/Transforms/KernelsToCpp.cpp

```diff
@@ -120,14 +154,37 @@ LogicalResult emitDispatchOpRegionAsCpp(DispatchOp origOp,
   IRMapping irMapper;
   funcBody->takeBody(region);
 
+  // Apply arith to emitc conversion first
+  {
+    ConversionTarget target(*module.getContext());
+    target.addLegalDialect<emitc::EmitCDialect>();
+    target.addIllegalDialect<arith::ArithDialect>();
```

Comment:
What does it mean to set arith to illegal?

Reply:
It means that no arith dialect ops should remain after emitc conversion.